code/modules/medical/genetics/bioHolder.dm

```diff
@@ -1,4 +1,19 @@
-
+/* IMPORTANT NOTES ABOUT THE APPEARANCE HOLDER
+   Hi, auntie Lagg here to explain you how the Appearance Holder makes people look different!
+	 Just about everything visual that isnt already defined in a specific bodypart is defined here
+	 Skin tone, hair / detail colors and styles
+
+	 If your mob uses a different skintone than the one defined by client prefs,
+	 swap the mob_appearance_flags HAS_HUMAN_SKINTONE with HAS_SPECIAL_SKINTONE
+	 and set the skintone in s_tone_special. The body renderer will handle the rest!
+	 This way, if we for whatever reason return to something with human skintone
+	 We can just swap the flags and we'll use our old skintone!
+
+	 more info about appearance and color flags in _std/macrow/_appearace.dm
```

Comment:
I'm just going to remove that comment chunk, most of it is out of date because I got the bright idea to write documentation for things that didnt exist.